en/docs/index.md

```diff
@@ -7,14 +7,18 @@ template: templates/single-column.html
     <header>
         <h1>Welcome to the WSO2 Security and Compliance Documentation!</h1>
     </header>
-    <div class="md-main .md-content" style="float:left; width: 45%;  text-align:justify; max-height:100%; ">
-        <p style="margin-bottom:10%">
-        Security is our top priority and we proactively look for ways to improve the security of our products, services, and infrastructure. Our security program is transparent so you can be confident when using our products and services, and infrastructure for mission-critical projects.      
+    <div class="md-main .md-content" style="float:center; width: 45%;  text-align:justify; max-height:100%; ">
+        <p>
+        Security is our top priority and we proactively look for ways to improve the security of our products, services,
+        and infrastructure. Our security program is transparent so you can be confident when using our products and 
+        services, and infrastructure for mission-critical projects. 
```

Comment:
```suggestion
        services and infrastructure for mission-critical projects. 
```

Reply:
Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8